### ISSUE TITLE
Merge bilingual and CQL features from upstream

### ISSUE THREAD

**Issue by jvanulde** [OpenDRR/pygeoapi#7]
Once https://github.com/geopython/pygeoapi/pull/670 is merged into main on upstream branch, update OpenDRR/pygeoapi and rebuild container on DockerHub.

**Comment by francbartoli** [comment]
@jvanulde please be aware that https://github.com/geopython/pygeoapi/pull/723 is your target now

**Comment by jvanulde** [comment]
https://github.com/OpenDRR/pygeoapi/commit/1294092645adb0724737fc64000c4aaca53bac57
